src/lib/Modes.ts

```diff
@@ -67,7 +76,7 @@ export default class Modes {
 				if (
 					ratio >= 0 &&
 					this.library.interactivity.mouseStatus ===
-						MouseInteractivityStatus.MOUSEMOVE
+					MouseInteractivityStatus.MOUSEMOVE
```

Comment:
Sorry for that. I thought I had disabled prettier. Everything below line 79 (including this line) is just prettier going nuts.